VideoLocker/src/main/java/org/edx/mobile/view/CourseDiscussionResponsesFragment.java

```diff
@@ -148,4 +123,90 @@ public void onClickAddComment(@NonNull DiscussionComment comment) {
     public void onClickViewComments(@NonNull DiscussionComment comment) {
         router.showCourseDiscussionComments(getActivity(), comment, discussionThread.isClosed());
     }
+
+    private class CommentsLoader {
+
+        private InfiniteScrollUtils.PageLoadCallback<DiscussionComment> callback;
+
+        private GetCommentListTask getCommentListTask;
+        private GetCommentListTask getEndorsedListTask;
+
+        private ThreadComments unendorsedComments;
+        private int nextPage = 1;
+        private boolean isQuestionTypeThread = false;
+        private boolean isEndorsedFetched = false;
```

Comment:
I guess the `isEndorsedFetched` variable should be grouped with `unendorsedComments` since these are both linked using the same logic.